view.go

```diff
@@ -425,6 +430,12 @@ func (v *View) Evict(key string) error {
 
 // Recovered returns true when the view has caught up with events from kafka.
 func (v *View) Recovered() bool {
+	// no partitions --> never recover
+	// Otherwise we might mask errors of initializing the view
+	if len(v.partitions) == 0 {
```

Comment:
possible bugfix: if views were empty due to errors, they might have appeared to be recovered.